ingest-app/src/cmr/ingest/api/core.clj

```diff
@@ -170,16 +170,20 @@
     {}
     {:threshold 1000}))
 
+(defn get-user-id
+  "Get user id based on context and headers"
+  [context headers]
+  (if-let [user-id (get headers "user-id")]
+    user-id
+    (when-let [token (get headers "echo-token")]
```

Comment:
I prefer `transmit-config/token-header` over `"echo-token"`. Not sure why you changed it.

Reply:
Done. Added user-id-header in transmit-config.